apps/bestofjs-nextjs/src/app/api/og/projects/[slug]/route.tsx

```diff
@@ -24,18 +25,29 @@ export async function GET(_req: Request, { params: { slug } }: Context) {
 
   return generateImageResponse(
     <ImageLayout>
-      <Box style={{ alignItems: "center", gap: 48 }}>
+      <Box
+        style={{
```

Comment:
I'm curious we use style objects instead of `cn()` for the OG feature.

Reply:
> I'm curious we use style objects instead of `cn()` for the OG feature.

OG images are generated by a specific rendering engine called https://github.com/vercel/satori

It looks like our usual React components but it's not.

However, if you look at their playground https://og-playground.vercel.app/ , there is **experimental** support for TailwindCSS syntax: but it's not based on the usual `className` property, it uses a very specific `tw` property.

We could use it, since we don't any of the constraints of web, I find inline styles and the object syntax more readable but I don't have a strong opinion, we could try this approach.

The example from their playground:

```html
<div tw="flex flex-col w-full h-full items-center justify-center bg-white">
  <div tw="bg-gray-50 flex w-full">
    <div tw="flex flex-col md:flex-row w-full py-12 px-4 md:items-center justify-between p-8">
      <h2 tw="flex flex-col text-3xl sm:text-4xl font-bold tracking-tight text-gray-900 text-left">
        <span>Ready to dive in?</span>
        <span tw="text-indigo-600">Start your free trial today.</span>
      </h2>
      <div tw="mt-8 flex md:mt-0">
        <div tw="flex rounded-md shadow">
          <a tw="flex items-center justify-center rounded-md border border-transparent bg-indigo-600 px-5 py-3 text-base font-medium text-white">Get started</a>
        </div>
        <div tw="ml-3 flex rounded-md shadow">
          <a tw="flex items-center justify-center rounded-md border border-transparent bg-white px-5 py-3 text-base font-medium text-indigo-600">Learn more</a>
        </div>
      </div>
    </div>
  </div>
</div>
```

Reply:
Chiming in, I think the inline styling is nice here for the readability. But I could also see that using the TailwindCSS syntax could be more consistent and slightly less code.

Reply:
Ok I figured it was a limitation I wasn't aware of.

I think we should wait until tailwind support is more mature, the OG styling is pretty self contained and short.